templates/careers/sales.html

```diff
@@ -57,23 +57,23 @@ <h3 class="p-heading--2">Projects you'll work on</h3 class="p-heading--2">
   <div class="p-strip is-shallow u-no-padding--top">
     <div class="row">
       <div class="col-6">
-        <h3 class="p-heading--2">Sales at Canonical</h3 class="p-heading--2">
+        <h3 class="p-heading--2">Projects you'll work on</h3>
       </div>
       <div class="col-6">
-          <p>Enterprises love Ubuntu and Enterprises deploy Open Source. Become the trusted advisor and work on multi-million dollar contracts to build, support and manage open source projects to drive innovation.</p>
+        <p>The key focus of our sales team is helping customers use and deploy open source software at scale and ensuring it's maintained and secured. You will work closely with some of the world’s leading brands and organisations and help them consume high-quality open source software to achieve their ambitions. From self-driving cars to critical government systems, we take pride in helping our customers shape the future.</p>
       </div>
     </div>
   </div>
   <div class="p-strip is-shallow u-no-padding--top">
     <hr class="is-fixed-width u-no-margin--bottom">
     <div class="row">
       <div class="col-6">
-        <h3 class="p-heading--2">Who you are</h3 class="p-heading--2">
+        <h3 class="p-heading--2">Who you are</h3>
       </div>
       <div class="col-6">
         <p>Your approach is consultative, you are results and goal-oriented, and you know how to invest your time to maximise the benefit for customers and the company.</p>
-        <p>You build long term relationships, and you always promote the approach that is in the customers best interests, regardless of short term consequences. You are interested in the state of the art, you love to represent a company that knows how to transform customer operations, and you are keen to see new customers benefit from that knowledge.</p>
-        <p>You are organised, persistent, friendly and hard-working. You are effective in a digital world, keeping track of your plans and your progress, collaborating with colleagues to deliver for your customers. You are professionally competitive &ndash; respectful of the competition and determined to improve until you win regularly.</p>
+        <p>You build long-term relationships, and you always promote the approach that is in the customer’s best interests, regardless of short-term consequences. You are interested in the state of the art, you love to represent a company that knows how to transform customer operations, and you are keen to see new customers benefit from that knowledge.</p>
```

Comment:
This apostrophe looks different from the one in `Projects you'll work on`? I suppose that's the right one